can_tools/scrapers/official/CT/ct_state.py

```diff
@@ -0,0 +1,164 @@
+from typing import Any
+
+import pandas as pd
+import us
+import requests
+
+from can_tools.scrapers.base import CMU, DatasetBase
+from can_tools.scrapers.official.base import SODA
+
+
+class CTCountyDeathHospitalizations(SODA, DatasetBase):
+    baseurl = "https://data.ct.gov/"
+    has_location = False
+    location_type = "county"
+    cdh_id = "bfnu-rgqt"
+    state_fips = int(us.states.lookup("Connecticut").fips)
+    source = "https://data.ct.gov/resource/bfnu-rgqt.json"
+
+    def fetch(self):
+        cdh = self.get_dataset(self.cdh_id)
+
+        return cdh
+
+    def normalize(self, data) -> pd.DataFrame:
+        # fips_df = pd.DataFrame.from_dict(self.state_fips)
+        cdh_rename = {"dateupdated": "dt", "county": "location_name"}
+        cdh = data.rename(columns=cdh_rename)
+        cdh["dt"] = pd.to_datetime(cdh["dt"])
+        cdh = cdh.reindex(columns=cdh.columns)
+        for c in [
+            c for c in cdh_rename.values() if (c != "dt") and (c != "location_name")
+        ]:
+            cdh[c] = pd.to_numeric(cdh.loc[:, c])
+        crename = {
+            "totalcases": CMU(
+                category="cases", measurement="cumulative", unit="people"
+            ),
+            "totaldeaths": CMU(
+                category="deaths", measurement="cumulative", unit="people"
+            ),
+            "hospitalization": CMU(
+                category="hospital_beds_in_use_covid",
+                measurement="current",
+                unit="beds",
+            ),
+        }
+        out = cdh.melt(
+            id_vars=["dt", "location_name"], value_vars=crename.keys()
+        ).dropna()
+
+        out["vintage"] = self._retrieve_vintage()
+        out = self.extract_CMU(out, crename)
+        return out
+
+
+class CTCountyTests(SODA, DatasetBase):
+    baseurl = "https://data.ct.gov/"
+    test_id = "qfkt-uahj"
+    state_fips = int(us.states.lookup("Connecticut").fips)
+    has_location = False
+    location_type = "county"
+    source = "https://data.ct.gov/resource/qfkt-uahj.json"
+
+    def fetch(self):
+        tests = self.get_dataset(self.test_id)
+
+        return tests
+
+    def normalize(self, data: Any) -> pd.DataFrame:
+        # Get raw dataframe
+        tests_rename = {"date": "dt", "county": "location_name"}
+        tests = data.rename(columns=tests_rename)
+
+        tests["dt"] = pd.to_datetime(tests["dt"])
+
+        tests = tests.reindex(columns=tests.columns)
+        for c in [c for c in tests_rename.values() if "number" in c]:
```

Comment:
Same two comments here:

- Looks like the `.reindex` call is reduntant
- Loop iterator seems to be empty